app/views/key_pairs/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<% title _("Key pairs for: %s") % @compute_resource.name %>
+
+<table class="<%= table_css_classes 'table-fixed' %>">
+  <thead>
+  <tr>
+    <th width="6%"><%= _('Active key') %></th>
+    <th width="40%"><%= _('Name') %></th>
+    <th width="40%"><%= _('Fingerprint') %></th>
+    <th width="100px"><%= _('Actions') %></th>
+  </tr>
+  </thead>
+  <tbody>
+  <% @key_pairs.each do |key_pair| %>
```

Comment:
what if there are no key pairs(deleted by mistake - e.g. in ec2 console)? I think it make sense to allow to recreate the key? if you go that path, maybe consider adding patternfly empty state ?